### PR TITLE
15 minute time intervals

### PR DESCRIPTION
**Pull request by CNichole726**
Date picker and time picker are functioning, time with 15 minute intervals. Jessica has an idea to save the date as a Date object that she is going to try separately. For now we are converting them to strings.

if you get that error we saw on imports you can use this `npm install --save @ng-bootstrap/ng-bootstrap`

### CODE REVIEW

File: src/app/event-form/event-form.component.html [hunk 1]
```diff
@@ -125,30 +125,35 @@
...
-            <input type="date" [(ngModel)]="event.date" 
-              class="form-control"
+            <ngb-datepicker [(ngModel)]="eventDate" 
               id="date" 
               name="date"
               placeholder="Enter Event Date"
               required #date="ngModel">
-
+            </ngb-datepicker>
+            <!-- <button class="btn btn-sm btn-outline-primary mr-2" (click)="selectToday()">Select Today</button>
```
Comment (lfallert) [inline src/app/event-form/event-form.component.html:137]: do we need this commented code?

File: src/app/event-form/event-form.component.ts [hunk 2]
```diff
@@ -13,11 +15,30 @@ export class EventFormComponent implements OnInit {
   event: Event;
   eventCost: boolean = true;
   eventCostNum: number;
+  model: NgbDateStruct;
+  eventDate: {year:number, month: number, day: number};
+  eventTime: {hour:number, minute:number};
+  meridian = true;
+  minuteStep = 15;
+  // eventDate: Object;
```
Comment (lfallert) [inline src/app/event-form/event-form.component.ts:24]: do we need this commented code?

**Review by lfallert — APPROVED** [review]
functioning now with minutes saved as "00" when appropriate. We should look into changing the format to nonmilitary time, in a later feature. Good job!

**Review by aeydie — APPROVED** [review]
Confirmed commented out text was wanted, non military time will be figured out later given time, everything functions when tested and code looks good! Great job!

**Pull request merged**